frontend/src/pages/TeamFeedPage/TeamFeedPage.tsx

```diff
@@ -120,7 +120,7 @@ const TeamFeedPage = (props: TeamFeedPageProps) => {
                 : '여기에 채팅을 입력하세요. \n\nShift + Enter로 새 행을 추가합니다.'
             }
             maxLength={10000}
-            autoFocus
+            autoFocus={isMobile ? false : true}
```

Comment:
```suggestion
            autoFocus={!isMobile}
```

혹시 이렇게 바꾸는 게 가독성 면에서 안 좋을까? `isMobile` 자체가 `boolean` 타입이라서 그대로 대입해도 온전하게 작동할 것 같아 보여서!

Reply:
오 좋아좋아 간단해서 가독성이 더 좋네